doc/source/development/rfc/rfc102_embedded_resources.rst

```diff
@@ -0,0 +1,166 @@
+.. _rfc-102:
+
+===================================================================
+RFC 102: Embedding resource files into libgdal
+===================================================================
+
+============== =============================================
+Author:        Even Rouault
+Contact:       even.rouault @ spatialys.com
+Started:       2024-Oct-01
+Status:        Draft
+Target:        GDAL 3.11
+============== =============================================
+
+Summary
+-------
+
+This RFC uses C23 ``#embed`` pre-processor directive, when available,
+to be able to optionally embed GDAL resource files directly into libgdal.
+
+A similar `PROJ RFC-8 <https://github.com/OSGeo/PROJ/pull/4274>`__ has been
+submitted for PROJ to embed its :file:`proj.db` and :file:`proj.ini` files.
+
+Motivation
+----------
+
+Some parts of GDAL core, but mostly drivers, depend on a number of resource
+files for correct execution. Locating those resource files on the filesystem
+can be painful in some use cases of GDAL, that involve relocating the GDAL
+binary at installation time. One such case could be the GDAL embedded in Rasterio
+or Fiona binary wheels where :config:`GDAL_DATA` must be correctly set currently.
```

Comment:
Does replacing "must be correctly set currently" with "must be set to the directory of the resource files" work?

Reply:
thanks, adopted